src/components/Repository.js

```diff
@@ -98,8 +96,6 @@ function Repository({match}) {
       .finally(() => setIsForkLoading(false));
   };
 
-  const user = getUserFromJwt(Config.auth);
```

Comment:
user is now a props of Repository

Reply:
FYI, @filiptronicek. This should make you able to make your change without failing or updating the test.

Reply:
Awesome!